Console/Command/AbstractStoreCommand.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Console\Command;
+
+use Algolia\AlgoliaSearch\Service\StoreNameFetcher;
+use Magento\Framework\App\Area;
+use Magento\Framework\App\State;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+abstract class AbstractStoreCommand extends Command
+{
+    protected const STORE_ARGUMENT = 'store';
```

Comment:
Having `store` as parameter can be confusing since we are expecting a store id. WDYT about renaming it to `store_id` to be clearer with the expected type ?

Reply:
J'aime ça!